src/styled/react-components/react-components.ts

```diff
@@ -12,4 +12,29 @@ export const HtmlFieldWrapper = styled(Box, {
   height: "auto"
 }));
 
-export default HtmlFieldWrapper;
\ No newline at end of file
+/**
+ * Styled url field component
+ */
+export const UrlFieldWrapper = styled(Input, {
+  label: "url-field-wrapper"
+})(() => ({
+  flex: 1,
+  width: "80%",
+  height: "auto",
+  backgroundColor: "#fff"
+}));
+
+/**
+ * Styled text field component
+ */
+export const TextFieldWrapper = styled(Input, {
+  label: "text-field-wrapper"
+})(() => ({
+  flex: 1,
+  width: "100%",
+  height: "auto",
+  fontSize: "15px",
```

Comment:
It might be better to use em for font size if possible as em tends to be more responsive and scalable. As you used rem for the BorderRadius slightly below.

Reply:
Fixed